cypress/integration/visual-regression.spec.js

```diff
@@ -64,9 +64,9 @@ describe('Percy Snapshots', () => {
   })
 
   describe('Radio', () => {
```

Comment:
Hey, it's better to have one story containing both examples, like we have for buttons